### ISSUE TITLE
Lots of IPFS errors with Turkish wikipedia mirror (loading fine nonetheless)

### ISSUE THREAD

**Issue by JayBrown** [ipfs/distributed-wikipedia-mirror#39]
The IPFS mirror of the Turkish wikipedia (via IPNS) loads fine, but lots of errors are popping up:

```
22:01:30.949 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Mameluke_Flag.svg.png: Failed to get block for zb2rhm7VzayQudATK8W2u37aGhPmEFknTou6zQXxCSHj6JYte: context canceled gateway_handler.go:548
22:01:30.966 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Flag_of_Greece_(1822-1978).svg.png: Failed to get block for QmP7hqsgd8NX2aCRUS3oGbgmM6ZSWaW21j7eXZ1ba7FVTv: context canceled gateway_handler.go:548
22:01:30.966 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Coa_Croatia_Country_History_(Fojnica_Armorial).svg.png: Failed to get block for QmVfQXFGao22h7EvPuH66dakgj4FxkN98cuYNYFKr6bCoG: context canceled gateway_handler.go:548
22:01:30.967 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Coat_of_arms_of_Hungary_(historic_design).png: Failed to get block for zb2rhX18u3QKYQQqeJtfE7kjHdNWtj9MhmZMNKNGoLLeT2Vmn: context canceled gateway_handler.go:548
22:01:30.969 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Coa_Kastrioti_Family.svg.png: Failed to get block for QmVrXEXLDLgGtdA5avsnAsfEwEFWXv1gkqYutc338TP6W8: context canceled gateway_handler.go:548
22:01:30.975 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Flag_of_Turkey.svg.png: Failed to get block for QmNhRZWFoKfeuwbnpf35J9g1NzZCzSfHqPr1WiHqb8kSVf: context canceled gateway_handler.go:548
22:02:10.846 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Botanik_Parkı.jpg: Failed to get block for QmbemscoX5aXoNXRTTJ1adbysDTTZFMbt6YwZVynkn2dhL: context canceled gateway_handler.go:548
22:02:10.850 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Harikalar_Diyari_Gulliver_Lilliputians_06034_nevit.jpg: Failed to get block for QmPvPmfSc9GcDriwZA99AD3MHqKbpTVTKiuU2KVJBdTtdr: context canceled gateway_handler.go:548
22:02:10.856 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Çayyolu_Metro_19.JPG: Failed to get block for QmXxcZjuJHq2qnxGEedooW1WNdHtzL77UmwsxHSEBpoHHj: context canceled gateway_handler.go:548
22:02:10.857 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Ankara_Central_Station_2012_03.JPG: Failed to get block for QmRfdvNsndpiNEbNTuH1ZaDfrrMsa1WNDKMNoT7HBcXkzY: context canceled gateway_handler.go:548
22:02:10.880 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Ankara_road_map.png: Failed to get block for zb2rhgcucL97ksC69WTWyfp7SF7FQPkZi6T6gz79qxNaGM2Ca: context canceled gateway_handler.go:548
22:02:10.884 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Dirus-Roads_of_Ankara.svg.png: Failed to get block for zb2rhiF3XBBBvTDhYAhw7qgcEB1FRtCrATUgZmEJ38bxQ9tjv: context canceled gateway_handler.go:548
22:02:17.185 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Desc-20.png: Failed to get block for QmXu8c6yjTpxNgiqmBnEjXCGRNVCNXpFzy6QdBPdef7q8c: context canceled gateway_handler.go:548
22:02:17.185 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/BlankMapTurkeyProvinces.png: Failed to get block for QmSDKbXfco38V8CR7niButc81cogiW9m1QaXHaxpocBTvD: context canceled gateway_handler.go:548
22:02:17.218 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Mausoleo_Mevlana.jpg: Failed to get block for zb2rhmWtXQ19xRDHSnWtHCzLk6tfVmq3wFFkf5FbUWsksFrta: context canceled gateway_handler.go:548
22:02:17.222 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Dolmabahçe_Palace_(cropped).JPG: Failed to get block for QmUXuCuWnqYE82sCYNQnn35Kk4yHgkHSgeNdcgymdWc3xj: context canceled gateway_handler.go:548
22:02:17.267 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Mustafa_Kemal_Atatürk_.jpg: Failed to get block for Qmbw36E5GkUSfnZ2jida2ufKNSEejpXwCH1eUDhgZBdB23: context canceled gateway_handler.go:548
22:02:17.323 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Recep_Tayyip_Erdogan.PNG: Failed to get block for QmaAUgvy4fEK76RdKkEVfYErR3LCF9iFbBqP4tui3kmRbE: context canceled gateway_handler.go:548
22:02:23.634 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Erdoğan_(2014_cumhurbaşkanlığı_seçim_logosu).jpg: Failed to get block for QmXqbHEuXywNnxQyygjZZpxVaaf2tWffrG7gUz6Tt4hTjS: context canceled gateway_handler.go:548
22:02:23.637 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Cari_islemler_dengesi.png: Failed to get block for QmRx4esjoYLmBV7nC4i1ZrqFSMAoqJ3bRgS5dK7fJ8LvdX: context canceled gateway_handler.go:548
22:02:23.637 ERROR core/serve: ipfs resolve -r /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/I/m/Family_photo_of_Council_of_ministers_of_Turkey_and_Spain.jpg: Failed to get block for QmQAe9RCdiuHunR6mmuMjYA2X8LGiWrYybPPLYgUy6Z57b: context canceled gateway_handler.go:548
22:02:23.704 ERROR commands/h: err: write tcp4 127.0.0.1:8080->127.0.0.1:57281: write: protocol wrong type for socket handler.go:288
22:04:34.318 ERROR core/serve: ipfs resolve -r /ipns/QmVH1VzGBydSfmNG7rmdDjAeBZ71UVeEahVbNpFQtwZK8W/I/m/c326d317eddef3ad3e6625e018a708e290a039f6.svg: Failed to get block for zb2rhXQVxo883MW1cS2mYa3wYSv2ZLiQTmbn2EFrUXa1DSZUi: context canceled gateway_handler.go:548
22:04:34.319 ERROR core/serve: ipfs resolve -r /ipns/QmVH1VzGBydSfmNG7rmdDjAeBZ71UVeEahVbNpFQtwZK8W/I/m/c5d0431ce231935522dc0cb52df7f2b406cdadc3.svg: Failed to get block for zb2rhfsuyFnUf9j4BJx27sNzNB31mYSkdDQHgEWCLB5vxaDq8: context canceled gateway_handler.go:548
22:04:34.320 ERROR core/serve: ipfs resolve -r /ipns/QmVH1VzGBydSfmNG7rmdDjAeBZ71UVeEahVbNpFQtwZK8W/I/m/e1d67495288eac0fa90d5bbcad7d9a343c15ad56.svg: Failed to get block for zb2rhkXbtbZ9HpyHJixHRs6ioKjktd1ZJCiGfTNAK29BS8hjb: context canceled gateway_handler.go:548
22:04:34.320 ERROR core/serve: ipfs resolve -r /ipns/QmVH1VzGBydSfmNG7rmdDjAeBZ71UVeEahVbNpFQtwZK8W/I/m/504dc030b18a6fcb9575b9c70b2d9314e86ece5e.svg: Failed to get block for QmemtQK1z99jDhqsHgnwihX3PjiWJpjHRA6PfXN6oCFeQV: context canceled gateway_handler.go:548
22:04:34.321 ERROR core/serve: ipfs resolve -r /ipns/QmVH1VzGBydSfmNG7rmdDjAeBZ71UVeEahVbNpFQtwZK8W/I/m/Yüzey_sınıflandırması.jpg: Failed to get block for zb2rhnA149yrwMBERK9q35ZosesLekqEDn5ExhRjZYZi1xUn6: context canceled gateway_handler.go:548
22:04:34.322 ERROR core/serve: ipfs resolve -r /ipns/QmVH1VzGBydSfmNG7rmdDjAeBZ71UVeEahVbNpFQtwZK8W/I/m/21de672b1953817ed423e8f4c008498a81341292.svg: Failed to get block for zb2rhaDTRHNvVM6PW88ih4J7euvotR34SozzZ1dvDdRbMe6sj: context canceled gateway_handler.go:548
```

**Comment by lgierth** [comment]
"context canceled" means that the request which triggered the block-get was cancelled. These are not really errors in the sense that something failed. We should look into making them NOTICE level (or even DEBUG).

**Comment by lidel** [comment]
(assuming this is fixed in go-ipfs 0.4.22+)
